09-Multiparticle-Collision-Dynamics/01-Pressure-Driven-Flow.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>For style consistency with other tutorial, replace the matplotlib import with:</p>

<pre class="ql-syntax" spellcheck="false">%matplotlib inline
matplotlib.style.use('ggplot')
import matplotlib_inline

matplotlib_inline.backend_inline.set_matplotlib_formats('svg')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>